data/debug-cli/docker_debug.yaml

```
@@ -0,0 +1,166 @@
+command: docker debug
+short: Get a shell into any container or image.
+long: |-
+  > **Note**
+  >
+  > Docker Debug requires a [Pro, Team, or Business subcription](/subscription/details/).
+  > You must [sign in](/desktop/get-started/) to use this command.
+
+  Get an enhanced shell with additional tools into any container or image.
+  
+  The shell contains the following builtin commands:
+  - `install [tool1] [tool2]`: Add Nix packages from: https://search.nixos.org/packages.
+  - `uninstall [tool1] [tool2]`: Uninstall NixOS package(s).
+  - `entrypoint`: Print/lint/run the entrypoint.
+  - `builtins`: Show builtin commands.
+
+  For images and stopped containers, all changes are discarded when leaving the
+  shell. At no point, do changes affect the actual image or container. When
+  accessing running or paused containers, all filesystem changes are directly
+  visible to the container. The `/nix` directory is never visible to the actual
+  image or container.
+  
+usage: debug [OPTIONS] {CONTAINER|IMAGE}
+pname: docker
+plink: docker.yaml
+options:
+    - option: shell
+      value_type: shell
+      default_value: "auto"
+      description: "Select a shell. Supported: `bash`, `fish`, `zsh`, `auto`."
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: command
+      shorthand: c
+      value_type: string
+      default_value: false
+      description: Evaluate the specified commands instead, passing additional positional arguments through $argv.
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: host
+      value_type: string
+      default_value: false
+      description: "Daemon docker socket to connect to. E.g.: `ssh://root@example.org`, `unix:///some/path/docker.sock`"
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+    - option: version
+      value_type: bool
+      default_value: "false"
+      description: Display version of the docker-debug plugin
+      deprecated: false
+      hidden: false
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+deprecated: false
+experimental: false
+experimentalcli: false
+kubernetes: false
+swarm: false
+examples: |-
+  ### Example 1: Get a shell into any image
+  This example pulls an nginx image and gets a shell into it without ever running a container.
+  
+  ```console
+  $ docker debug nginx
+  
+  Builtin commands:
+  - install [tool1] [tool2] ...    Add Nix packages from: https://search.nixos.org/packages
+  - uninstall [tool1] [tool2] ...  Uninstall NixOS package(s).
+  - entrypoint                     Print/lint/run the entrypoint.
```

What does print/lint/run refer to? Does the `entrypoint` command when invoked in an interactive debug shell only display the container's entrypoint, or does it do more?

From the output below it seems a lint of the entrypoint happens on starting the debug shell.

```suggestion
  - entrypoint                     Print/lint/run the entrypoint.
```

---

Added examples for these.